docs/plugins.md

```diff
@@ -248,10 +248,10 @@ vs
 
 This also works with scoped packages
 
-```js
+```json
 {
-  presets: ["@org/babel-preset-name"], // actual package
-  presets: ["@org/name"] // shorthand name
+  "presets": ["@org/babel-preset-name"], // actual package
```

Comment:
I split the code blocks so it's consistent with the rest of the paragraph.

<img width="933" alt="screen shot 2016-12-11 at 9 20 32 am" src="https://cloud.githubusercontent.com/assets/927264/21081816/505c35f2-bf83-11e6-98de-b34575c0307d.png">